website/docs/core-concepts/events.md

```diff
@@ -4,4 +4,59 @@ sidebar_position: 9
 
 # Events
 
-## Coming Soon
+Events define when to trigger a notification for a webhook.
```

Comment:
Add examples on how and when you could trigger events come in handy:

- [ ] A webhook to trigger a static site to build based on updates
- [ ] Notifications on Slack & Google Chat when a certain action is made

Add following pages on `How-to Guides`
- [ ] Trigger build on Static Sites. Use Netlify, Render, Vercel as sub-sections. Check with @fyrepenguin for details
- [ ] Example for notification on Slack
- [ ] Example for notification on Google Chat